hack/make-release-artifacts.sh

```diff
@@ -92,6 +92,11 @@ mk_istio_manifests() {
 mk_kustomize_base() {
   for file_to_copy in ./kubernetes-manifests/*.yaml
   do
+    # Don't copy kustomization.yaml.
+    if [[ $file_to_copy == "./kubernetes-manifests/kustomization.yaml" ]]; then
+      continue
+    fi
+
```

Comment:
I just added this change into `hack/make-release-artifacts.sh`.
Context: Today, when a new release of Online Boutique is created `kubernetes-manifests/kustomization.yaml` is copied over to `kustomize/base/kustomization.yaml`. This should not be the case since the two files are meant to look different.
For instance, `kustomize/base/kustomization.yaml` should *not* include:
```
# components:
```
or
```
# - loadgenerator.yaml # During development, the loadgenerator module inside skaffold.yaml will be used.
```